workflow.cwl

```diff
@@ -38,7 +38,7 @@ steps:
         source: "#submitterUploadSynId"
       # TODO: replace `valueFrom` with the admin user ID or admin team ID
       - id: principalid
-        valueFrom: "3379097"
+        valueFrom: "3441739"
```

Comment:
Maybe switch this to the Organizers team ID, in case Olga and Nick want to check the log files.

Reply:
this makes more sense!